streamz/tests/test_core.py

```diff
@@ -931,3 +931,28 @@ def slow_write(x):
 
 if sys.version_info >= (3, 5):
     from streamz.tests.py3_test_core import *  # noqa
+
+
+def test_clear():
+    s = Stream()
+    # increment
+    def acc1(x1, x2):
+        return x1 + x2
+    from streamz.core import ClearMSG, IgnoreMSG
+
+    clear_msg = ClearMSG()
+    ignore_msg = IgnoreMSG()
+
+    sout = s.accumulate(acc1)
```

Comment:
What would happen if you had a map before the accumulate?  Does map (or all other stream operations) also need to know what to do with ClearMSGs?

Reply:
I would imagine that all `update` methods will need a cutout for messages.